ocpp1.6/charge_point.go

```diff
@@ -247,21 +274,26 @@ func (cp *chargePoint) Start(centralSystemUrl string) error {
 
 func (cp *chargePoint) Stop() {
 	cp.client.Stop()
+
+	if cp.errC != nil {
```

Comment:
Can you copy the same block also to the Central System's `Stop`?

Reply:
Where is that CS stop method?

Reply:
Oh nvm, there is no wrapper function for that yet. I have a ToDo for that, so I'll add it in a later PR.